src/data/mv_polynomial/basic.lean

```diff
@@ -981,4 +981,23 @@ eval₂_hom_eq_zero _ _ _ h
 end aeval
 
 end comm_semiring
+
+section comm_ring
+variables [comm_ring R]
+
+instance : ring (mv_polynomial σ R) := add_monoid_algebra.ring
+
+@[simp] lemma coeff_neg (p : mv_polynomial σ R) (s) : coeff s (-p) = -coeff s p := rfl
+
+@[simp]
+lemma coeff_sub (p q : mv_polynomial σ R) (n : ℕ) : coeff s (p - q) = coeff s p - coeff s q := rfl
+
+@[simp] lemma monomial_neg (a : R) : monomial s (-a) = -(monomial s a) :=
+by rw [eq_neg_iff_add_eq_zero, monomial_add, neg_add_self, monomial, finsupp.single_zero]
```

Comment:
This is the only lemma that isn't a duplicate - can you move it to `comm_ring.lean` which is where the other lemmas are?

Reply:
Ok, done. Sorry, I didn't see the comm_ring.lean file and assumed there should be a comm_ring section in basic.lean.